docs/src/appendix/snforge-library/fs/file_parser.md

```diff
@@ -0,0 +1,47 @@
+# `FileParser<T>`
```

Comment:
I think these files could benefit from a very brief (one, max two sentence) descriptions of what they document.

Reply:
Added